### PR TITLE
DX-1773 EDP Adding PORTAL_DISABLE_CSRF_CHECK config

### PR DESCRIPTION
**Pull request by tbuchaillot**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>
https://tyktech.atlassian.net/browse/DX-1773

Added this undocumented config option for EDP
---

### Preview Link

<br>
https://deploy-preview-5744--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-enterprise-developer-portal/deploy/configuration/#portal_disable_csrf_check
### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

___

### **PR Type**
documentation

___

### **Description**
- Added documentation for a new configuration option `PORTAL_DISABLE_CSRF_CHECK` in the Tyk Enterprise Developer Portal. This option allows users to disable CSRF protection for all routes, which should only be done in development environments or when alternative security measures are in place.
- Included details on the configuration type and its implications for security.
- Made minor formatting adjustments to improve readability.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>configuration.md</strong><dd><code>Document new configuration option for disabling CSRF checks</code></dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-enterprise-developer-portal/deploy/configuration.md

<li>Added documentation for <code>PORTAL_DISABLE_CSRF_CHECK</code> configuration.<br> <li> Described the purpose and usage of disabling CSRF protection.<br> <li> Minor formatting adjustments.<br>

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5744/files#diff-bff942701e76bae5b357af2a798c59862dff033909d36d1d2bba2d878add91dd">+6/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/deploy/configuration.md [hunk 1]
```diff
@@ -656,5 +661,4 @@ PORTAL_DATABASE_CONNECTIONSTRING="admin:secr3t@(localhost:3308)/portal?charset=u
 PORTAL_DATABASE_ENABLELOGS=true
 PORTAL_DATABASE_MAXRETRIES=3
 PORTAL_DATABASE_RETRYDELAY=5000
-PORTAL_TIB_ENABLED=true
-```
```
Comment (sharadregoti) [inline tyk-docs/content/product-stack/tyk-enterprise-developer-portal/deploy/configuration.md]: Why remove these backticks?

**Review by sharadregoti — CHANGES_REQUESTED** [review]

File: tyk-docs/content/product-stack/tyk-enterprise-developer-portal/deploy/configuration.md (hunk 1 above)
Comment (tbuchaillot) [reply tyk-docs/content/product-stack/tyk-enterprise-developer-portal/deploy/configuration.md]: whoop!

Comment (tbuchaillot) [reply tyk-docs/content/product-stack/tyk-enterprise-developer-portal/deploy/configuration.md]: added back

**Review by sharadregoti — APPROVED** [review]

**Pull request merged**